### ISSUE TITLE
Copying files into container with relative path fails unless work_dir is specified

### ISSUE THREAD

**Issue by reitzig** [reitzig/container-do#14]
With image `non-root-workdir` built from:

```Dockerfile
FROM ubuntu
WORKDIR /work
```

```toml
# ContainerDo.toml
[container]
image = "non-root-workdir"

[[copy.setup]]
files = ["foo"]
to = "some/dir/"
```

**Expected:** Container starts and file `some/dir/foo` is present in the container working directory.

**Actual:** 
```
no such directory
exit status 1
```

**Workaround:** Explicitly set `container.work_dir`.

**Comment by reitzig** [comment]
**Analysis:** `container.work_dir` is optional. During `CreateContainer`, we account for that (and `docker inspect` do determine `WorkingDir`) -- but we never update the config. Later, `CopyFilesTo` assembles destination paths using an empty string as working directory.

**Comment by reitzig** [comment]
Fixed. Will be released with 1.2.1.
